server/odc-service/src/main/java/com/oceanbase/odc/service/session/util/SchemaExtractor.java

```diff
@@ -203,6 +226,21 @@ public RelationFactor visitDatabase_factor(Database_factorContext ctx) {
         public RelationFactor visitCreate_database_stmt(Create_database_stmtContext ctx) {
             return null;
         }
+
+        @Override
+        public RelationFactor visitSimple_func_expr(Simple_func_exprContext ctx) {
```

Comment:
this ast node is not correct, there are much more other node that can be referenced to function accessing~

Reply:
we'd better visit bit_expr node and use MySQLExpressionFactory to judge if this node is a function calling~

Reply:
Done!